perllib/FixMyStreet/App/Form/Waste/Bulky.pm

```diff
@@ -48,7 +48,17 @@ has_page summary => (
     fields => ['submit', 'tandc'],
     title => 'Submit collection booking',
     template => 'waste/bulky/summary.html',
+    next => 'payment',
+);
+
+has_page payment => ( # XXX need to actually take payment
+    title => 'Payment successful',
+    fields => [ 'continue' ],
```

Comment:
Not sure why, but whether or not I have rebased this branch onto the most recent master, I get a 'category: Please choose a category' error when I try to submit the payment page.

Reply:
You might need to add a 'Bulky collection' contact to your Peterborough body.